client/views/signin/non-hosted.hbs

```diff
@@ -0,0 +1,31 @@
+{{> logo}}
+<header>
+    <h1 class="text-center">Non-Hosted Sign In</h1>
+</header>
+
+<form method="post" action="/signin-non-hosted">
+    <fieldset>
+        <label>Username</label>
+        <input 
+            name="username"
+            type="text" 
+            pattern=".{1,}"
+            class="input-text fullwidth"
+            value="{{username}}" 
+            placeholder=""
+            required />
+    </fieldset>
+    <fieldset>
+        <label>Password</label>
```

Comment:
My web-fu is rusty.  This is just presenting a password-entry field, right?  (i.e., it's not using "password" as a static value for the password, right?)

Reply:
yep, it's a text input of type password for users to enter their password.